packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -2530,6 +2531,15 @@ export interface VideoStreamRendererViewState {
 // @public
 export const VideoTile: (props: VideoTileProps) => JSX.Element;
 
+// @beta
```

Comment:
Stable shouldn't export beta-only properties (you can update to only conditionally export this in the communication-react packlet index.ts)

Reply:
done